modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1078,6 +1274,11 @@ struct Infer: public cv::detail::KernelTag {
                 auto ii = inputs.at(input_name);
                 const auto & mm = std::get<1>(it);
                 non_const_prepm->emplace(input_name, configurePreProcInfo(ii, mm));
+
+                // NB: configure intput param for further preproc
+                if (uu.net_input_params.is_applicable(mm)) {
+                    const_cast<IEUnit::InputFrameDesc &>(uu.net_input_params).set_param(input_name, ii);
```

Comment:
It looks like both `for()` from `if()` and `else()` have the same code. We have `reshape` option for load and `non_const_prepm` for import. But `inputs` will look ugly. MB need to refactor this code.